crates/precompiles/src/tip_fee_manager/amm.rs

```diff
@@ -139,48 +107,10 @@ impl TipFeeManager {
         Ok(())
     }
 
-    /// Calculate validator token reserve minus pending swaps
+    /// Get validator token reserve
     fn get_effective_validator_reserve(&self, pool_id: B256) -> Result<U256> {
```

Comment:
nit: do we need this function anymore?

Reply:
i think the call in `calculate_burn_amount` is not needed now, was required to check pending swaps, but now we no longer have them with the immediate execution of fee swaps, removed `get_effective_validator_reserve` here https://github.com/tempoxyz/tempo/pull/1707/commits/e7ee926d8a665beea7b7c6f1424f5f22253fbf5c